clients/fides-js/src/lib/tcf.ts

```diff
@@ -119,37 +153,73 @@ export const generateTcString = async ({
 };
 
 /**
- * Call tcf() to configure Fides with tcf support (if tcf is enabled).
+ * Extract just the TC string from a FidesEvent. This will also remove parts of the
+ * TC string that we do not want to surface with our CMP API events, such as
+ * `vendors_disclosed` and our own AC string addition.
  */
-export const tcf = () => {
+const fidesEventToTcString = (event: FidesEvent) => {
+  const { fides_tc_string: cookieString } = event.detail;
+  if (cookieString) {
+    // Remove the AC portion which is separated by FIDES_SEPARATOR
+    const [tcString] = cookieString.split(FIDES_SEPARATOR);
+    // We only want to return the first part of the tcString, which is separated by '.'
+    // This means Publisher TC is not sent either, which is okay for now since we do not set it.
+    // However, if we do one day set it, we would have to decode the string and encode it again
+    // without vendor_consents
```

Comment:
this sneaks in a fix that is part of https://github.com/ethyca/fides/issues/4165 which says we should not send `vendors_disclosed` in the CMP API